src/jquery.eclipsefdn-api.js

```diff
@@ -1821,11 +1832,17 @@
             if (events.length > displayCount) {
               events = events.slice(0, displayCount);
             }
+            var defaultUrl = $parent.data("default-url");
             // post process the date to update date format
             for (var i = 0; i < events.length; i++) {
-                if (Date.now() > new Date(events[i]["end-date"])) {
+            	// remove registration completely if event is in the past or link is missing
+                if (Date.now() > new Date(events[i]["end-date"]) || !events[i]["registration"]) {
                 	delete events[i]["registration"];
                 }
+                // if default is set + info isn't set, default is set
+                if (defaultUrl && (events[i]["infoUrl"] === undefined || events[i]["infoUrl"] === "")) {
```

Comment:
What is the purpose of this default URL? We cant remove the link if we don't have a URL?